src/main/java/org/beanpod/switchboard/service/DecoderService.java

```diff
@@ -0,0 +1,9 @@
+package org.beanpod.switchboard.service;
+
+import org.beanpod.switchboard.dto.StreamDTO;
+
+import java.util.List;
+
+public interface DecoderService {
+  List<StreamDTO> getDecoderStreams(String decoderSerialNumber);
+}
```

Comment:
New layer, I like it!

Reply:
backend is like an onion
many layers
make you cry